tests/test_writers.py

```diff
@@ -191,3 +192,69 @@ def test_write_sssom_ontoportal_json(self):
             self.mapping_count,
             f"{path} has the wrong number of mappings.",
         )
+
+    # TODO: Determine assertions that should pass
+    # TODO: Fix failing assertions that should pass
+    # TODO: before merging, remove pass/fail comments at end of assertion lines
+    def test_missing_entries(self):
+        """Test to make sure that missing entires do not appear."""
+        df = pd.DataFrame(
+            [
+                {
+                    "subject_id": "Orphanet:58",
+                    "subject_label": "Alexander disease",
+                    "predicate_id": "skos:exactMatch",
+                    "object_id": "icd11.foundation:2023359698",
+                    "object_label": "Alexander disease",
+                    "mapping_justification": "semapv:UnspecifiedMatching",
+                }
+            ]
+        )
+        metadata = {
+            "creator_id": "orcid:0000-0002-2906-7319",
+            "curie_map": {
+                "MONDO": "http://purl.obolibrary.org/obo/MONDO_",
+                "Orphanet": "http://www.orpha.net/ORDO/Orphanet_",
+                "icd11.foundation": "http://id.who.int/icd/entity/",
+                "oboInOwl": "http://www.geneontology.org/formats/oboInOwl#",
+                "orcid": "https://orcid.org/",
+                "owl": "http://www.w3.org/2002/07/owl#",
+                "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+                "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+                "semapv": "https://w3id.org/semapv/",
+                "skos": "http://www.w3.org/2004/02/skos/core#",
+                "sssom": "https://w3id.org/sssom/",
+            },
+            "license": "http://w3id.org/sssom/license/unspecified",
+            "mapping_provider": "https://www.orpha.net/",
+        }
+
+        # When passing Converter
+        msdf: MappingSetDataFrame = MappingSetDataFrame(
+            converter=curies.Converter.from_prefix_map(metadata["curie_map"]),
+            df=df,
+            metadata=metadata,
+        )
+        self.assertIn("curie_map", msdf.metadata)  # pass
+        self.assertIn("icd11.foundation", msdf.metadata["curie_map"])  # pass
+        self.assertIn("icd11.foundation", msdf.prefix_map)  # pass
+        tmp_file = os.path.join(test_out_dir, "test_missing_entries.sssom.tsv")
```

Comment:
Is this file already there?

Reply:
I think I named this output file `test_missing_entries` because that is the name of this test method. I think that's the convention that was used for other output files for other methods, which makes sense.

I did just notice that this `tmp_file` declaration appeared in 2 different places in the method. I just reduced it to 1 and pushed an amended commit.

There is no other test that references this file.

Maybe you're asking, "is this supposed to be a static file that gets read into the test?", and if so, the answer is no. This `tmp_file` gets written in the test method based on the metadata and single mapping set row that I declare in the Python code. Then, after writing, it gets read and additional assertions are run.

Reply:
Sorry, that was I was asking and of course I see it now! Thank you!